packages/create-sitecore-jss/src/templates/nextjs/next.config.js

```diff
@@ -29,6 +29,14 @@ const nextConfig = {
   // Enable React Strict Mode
   reactStrictMode: true,
 
+  // enable image provider domains and device sizes
+  images: {
+    domains: ['jssConfig.sitecoreApiHost'],
+    // This array corresponds to the `sizes` property in the react Image.
```

Comment:
This comment doesn't seem accurate (according to our styleguide samples and [next.js docs](https://nextjs.org/docs/api-reference/next/image#sizes)). However, we should add a comment about these corresponding to our JSS image whitelisting config in the Sitecore config (similar to comments in the styleguide sample).